### PR TITLE
Added commands help, unknown. Added text in sendMessages. Some code i…

### PR DESCRIPTION
**Pull request by Toksel-pixel**
…mprove

### CODE REVIEW

File: Project/giftgenerator_bot/src/test/java/by/jrr/giftgenerator_bot/GiftgeneratorBotApplicationTests.java [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 
+import by.jrr.giftgenerator_bot.config.Mapper;
+import by.jrr.giftgenerator_bot.service.MessageService;
+import by.jrr.giftgenerator_bot.service.TelegramBot;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
-@SpringBootTest
-class GiftgeneratorBotApplicationTests {
+@SpringBootTest(classes = {TelegramBot.class, Mapper.class, MessageService.class})
```
Comment (Toksel-pixel) [inline Project/giftgenerator_bot/src/test/java/by/jrr/giftgenerator_bot/GiftgeneratorBotApplicationTests.java:9]: some code impprovement (from video)

File: Project/giftgenerator_bot/src/test/java/by/jrr/giftgenerator_bot/GiftgeneratorBotApplicationTests.java [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+public class GiftgeneratorBotApplicationTests {
```
Comment (Toksel-pixel) [inline Project/giftgenerator_bot/src/test/java/by/jrr/giftgenerator_bot/GiftgeneratorBotApplicationTests.java:10]: change this for code impprovement (from video)

File: Project/giftgenerator_bot/src/test/java/by/jrr/giftgenerator_bot/service/TelegramBotTest.java [hunk 3]
```diff
@@ -14,25 +15,25 @@
...
+
+class TelegramBotTest extends GiftgeneratorBotApplicationTests {
 
     @Autowired
     TelegramBot telegramBot;
     @Autowired
     ObjectMapper objectMapper;
 
-    @Test
+ //   @Test
```
Comment (Toksel-pixel) [inline Project/giftgenerator_bot/src/test/java/by/jrr/giftgenerator_bot/service/TelegramBotTest.java]: change this to a comment for start application

File: Project/giftgenerator_bot/src/test/java/by/jrr/giftgenerator_bot/service/TelegramBotTest.java [hunk 4]
```diff
@@ -14,25 +15,25 @@
...
     void onUpdateReceived() throws IOException {
         Update update = objectMapper.readValue(new File("src/test/resources/update.json"), Update.class);
         telegramBot.onUpdateReceived(update);
     }
 
-    @Test
+ //   @Test
```
Comment (Toksel-pixel) [inline Project/giftgenerator_bot/src/test/java/by/jrr/giftgenerator_bot/service/TelegramBotTest.java]: change this to a comment for start application 
and also have this test failed

**Review by Anuchek — APPROVED** [review]
cool

**Review by Parsleyka — APPROVED** [review]
I hope it will work

**Pull request merged**